docs/source/documentation/r_scripts.rst

```diff
@@ -0,0 +1,117 @@
+How-to use R scripts with Substra
+=================================
+
+SubstraFL is a library made for working in Python, but Substra is flexible enough to accommodate running tasks in other programming languages.
+This how-to guide explains how to run scripts written in R with Substra.
+This uses the low-level interface of Substra and requires writing more boilerplate code than using the high-level interface of SubstraFL.
+If you are not familiar with the Substra low-level library, you should read the
+:ref:`Substra introductory example <Titanic example>` first.
```

Comment:
Is it better to use the URL and remove the custom anchor in `examples/titanic_example`?
```suggestion
:doc:`Substra introductory example </auto_examples/titanic_example/run_titanic>` first.
```

Reply:
Thanks, I didn't manage to find the right URL, but it's better than a custom anchor, definitely